package.json

```diff
@@ -28,9 +29,10 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "d3-dsv": "^1.0.5",
+    "datavoyager": "^2.0.0-alpha.7",
+    "datavoyager-server": "git://github.com/vega/voyager-server.git",
```

Comment:
Can we make a release point to a version?   (Please add me as an NPM owner too :D)